Makefile

```
@@ -56,6 +59,28 @@ lint:
+# is only needed for the e2e database, which the Playwright stack expects to exist.
+# Idempotent, so it is safe to re-run.
+test-db:
+	@for db in open_hybrid_cloud_test open_hybrid_cloud_e2e; do \
+	  docker exec ohc-postgres psql -U postgres -tAc "SELECT 1 FROM pg_database WHERE datname='$$db'" | grep -q 1 \
+	    && echo "  exists  $$db" \
+	    || { docker exec ohc-postgres createdb -U postgres "$$db" && echo "  created $$db"; }; \
+	done
+
+# Drops the per-directory databases the backend suite created. They are cheap to
+# recreate (the schema is pushed on first run) and easy to forget about.
+test-db-prune:
```

checkmake (0.3.2)

[warning] Target body for "test-db-prune" exceeds allowed length of 5 lines (6).

(maxbodylength)